openedx/core/djangoapps/user_api/api.py

```diff
@@ -472,6 +475,88 @@ def _add_year_of_birth_field(self, form_desc, required=True):
             required=required
         )
 
+    def _add_profession_field(self, form_desc, required=True):
+        """Add a profession field to a form description.
+
+        Arguments:
+            form_desc: A form description
+
+        Keyword Arguments:
+            required (bool): Whether this field is required; defaults to True
+
+        """
+        # Translators: This label appears above a dropdown menu on the registration
+        # form used to select the user's profession
+        profession_label = _(u"Profession")
```

Comment:
Nit: Ditch the `u`. It's not necessary given that the string isn't actually Unicode.